CHANGELOG.md

```diff
@@ -43,6 +43,7 @@
 - #475 - Disallow subscribing to the same topic with multiple consumers
 - #485 - Setting shutdown_timeout to nil kills the app without waiting for anything
 - #487 - Make listeners as instances
+- #29 (Sidekiq backend) - Consumer class names must have the word "Consumer" in it in order to work
```

Comment:
`- #29 - Consumer class names must have the word "Consumer" in it in order to work (Sidekiq backend)`